crates/evm/src/evm/handler.rs

```diff
@@ -356,19 +356,16 @@ impl<SPEC: Spec, EXT: CitreaExternalExt, DB: Database> CitreaHandler<SPEC, EXT,
         let l1_fee_rate = context.external.l1_fee_rate();
         let l1_fee = U256::from(diff_size) * U256::from(l1_fee_rate);
         context.external.set_tx_info(TxInfo { diff_size, l1_fee });
-        if result.interpreter_result().is_ok() {
```

Comment:
in post execution output, we can't have txs that failed right?

assume we run a tx with wrong nonce, that doesnt come into the post execution output, right?

Reply:
Right, it fails far before execution, it's basically free for us